src/main/java/com/nylas/Maps.java

```diff
@@ -3,22 +3,22 @@
 import java.util.HashMap;
 import java.util.Map;
 
-class Maps {
+public class Maps {
 
-	static <K,V> Map<K,V> of(K key, V val) {
+	public static <K,V> Map<K,V> of(K key, V val) {
 		Map<K,V> map = new HashMap<>();
 		map.put(key, val);
 		return map;
 	}
-	
-	static <K,V> Map<K,V> of(K key, V val, K key2, V val2) {
+
+	public static <K,V> Map<K,V> of(K key, V val, K key2, V val2) {
```

Comment:
just curious, is this static method necessary? One can call the one in line 8, and add `key2` and `val2` with the `putIfNotNull` method below, no?

Reply:
Great call out, I actually haven't noticed that before. Pushing a fix for this now!